admin/page/ajax-edit-produk.php

```diff
@@ -0,0 +1,264 @@
+                } );
+            
+            $("#editor2").click(function (e) { 
+                e.preventDefault();
+                $("#specification2").val($("#editor2").html());
+                $(this).keyup(function () { 
+                    $("#specification2").val($("#editor2").html());
+                });
+            });
+        </script>
+        <?php
+    }
+    function add_deskripsi(){
+        global $deskripsi;
+        ?>
+        Deskripsi
+        <div id='toolbar-container4'>
+        </div>
+        <div id='editor4' class='border-gray'>
+            <?php include($deskripsi); ?>
+        </div>
+        <input type='hidden' name='deskripsi' id='deskripsi2' value="<?php include($deskripsi) ?>">
+        <script>
+            DecoupledEditor
+                .create( document.querySelector( '#editor4' ) )
+                .then( editor => {
+                    const toolbarContainer = document.querySelector( '#toolbar-container4' );
+
+                    toolbarContainer.appendChild( editor.ui.view.toolbar.element );
+                } )
+                .catch( error => {
+                    console.error( error );
+                } );
+            
+            $("#editor4").click(function (e) { 
+                e.preventDefault();
+                $("#deskripis2").val($("#editor4").html());
+                $(this).keyup(function () { 
+                    $("#deskripsi2").val($("#editor4").html());
+                });
+            });
+        </script>
+        <?php
+    }
+    function change_file(){
+        ?>
+        <div>
+            <label for='download'>File</label>
+            <input type='file' name='download' id='download' class='form-control-file ml-auto'>
+        </div>
+        <?php
+    }
+
+
+    ?>
+
+    <div class="form-inline">
+        <label>ID</label>
+        <input type="text" name="id_produk" class="form-control ml-auto" value="<?php echo $id ?>" readonly required>
+    </div>
+    <div class="form-inline">
+        <label>Nama Produk</label>
+        <input type="text" name="nama_produk" placeholder="Nama Produk" class="form-control ml-auto" value="<?php echo $nama_produk ?>" required>
+    </div>
+    <div class="form-inline">
+        <label>Jenis Produk</label>
+        <input type="text" name="jenis_produk" class="form-control ml-auto" value="<?php echo $jenis_produk ?>" readonly required>
+    </div>
+    <?php
+
+    switch ($jenis_produk) {
+        case 'Senjata':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Rifles"){echo "selected";} ?> >Rifles</option>
+                    <option <?php if($kategori=="Sniper Rifles"){echo "selected";} ?> >Sniper Rifles</option>
+                    <option <?php if($kategori=="Machine Guns"){echo "selected";} ?> >Machine Guns</option>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)